src/Compilers/CSharp/Portable/Symbols/Metadata/PE/SymbolFactory.cs

```diff
@@ -57,6 +57,11 @@ internal override bool IsVolatileModifierType(PEModuleSymbol moduleSymbol, TypeS
             return type.SpecialType == SpecialType.System_Runtime_CompilerServices_IsVolatile;
         }
 
+        internal override bool IsConstModifierType(PEModuleSymbol moduleSymbol, TypeSymbol type)
+        {
+            return type.ToDisplayString() == WellKnownTypes.GetMetadataName(WellKnownType.System_Runtime_CompilerServices_IsConst);
+        }
```

Comment:
Planning on fixing now.

---
In reply to: [117888944](https://github.com/dotnet/roslyn/pull/19658#discussion_r117888944) [](ancestors = 117888944,117561528)